content/sensu-go/5.14/api/federation.md

```diff
@@ -0,0 +1,408 @@
+---
+title: "Federation API"
+description: "The federation API controls federation of Sensu clusters. Here’s a reference for the federation API in Sensu Go, including examples. Read on for the full reference."
+version: "5.14"
+product: "Sensu Go"
+menu:
+  sensu-go-5.14:
+    parent: api
+---
+
+- [The `/replicators` API endpoint](#the-replicators-api-endpoint)
+	- [`/replicators` (GET)](#replicators-get)
+	- [`/replicators` (POST)](#replicators-post)
+- [The `/replicators/:replicator` API endpoint](#the-replicatorsreplicator-api-endpoint)
+	- [`/replicators/:replicator` (GET)](#replicatorsreplicator-get)
+  - [`/replicators/:replicator` (PUT)](#replicatorsreplicator-put)
+  - [`/replicators/:replicator` (DELETE)](#replicatorsreplicator-delete)
+
+## The `/replicators` API endpoint
+
+**LICENSED TIER**: Unlock the federation API in Sensu Go with a Sensu license. To activate your license, see the [getting started guide][1].
+
+_**NOTE**: The federation API is only accessible for users who have a cluster role that permits access to replication resources._
+
+### `/replicators` (GET)
+
+The `/replicators` API endpoint provides HTTP GET access to a list of replicators.
+
+#### EXAMPLE {#replicator-get-example}
+
+The following example demonstrates a request to the `/replicators` API, resulting in a list of replicators.
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/federation/v1/replicators -H "Authorization: Bearer $SENSU_TOKEN"
+[
```

Comment:
Need output/result examples for all endpoints

Reply:
Here's an example of an EtcdReplicator resource:
```
{
  "api_version": "federation/v1",
  "type": "EtcdReplicator",
  "metadata": {
    "name": "my_replicator"
  },
  "spec": {
    "insecure": true,
    "url": "http://127.0.0.1:3379",
    "api_version": "core/v2",
    "resource": "CheckConfig",
    "replication_interval_seconds": 5
  }
}
```

Reply:
Thank you. Is it OK to use this as the output example for all of the endpoints?

Reply:
Oops, I guess I never responded to this. Let's sync up on slack about this.